docs/docsite/rst/conf.py

```diff
@@ -90,10 +104,12 @@
 
 # A list of glob-style patterns that should be excluded when looking
 # for source files.
-exclude_patterns = [
+exclude_patterns = [] if tags.has('all') else [
     '2.10_index.rst',
     'ansible_index.rst',
     'core_index.rst',
```

Comment:
I think this is missing `'porting_guides/core_porting_guides',` and causing a ci failure with:

```
docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst:46:0: undefined-label: undefined label: 'windows_faq_server2008'
```

Reply:
https://github.com/ansible/ansible-documentation/pull/431/files#diff-8856b564ec7aee81e8229080c2f80c080fda52b748dac59af0337ae2f3688404L97

Reply:
Hmm. Although I see you added this in line 170 so maybe I should refrain from commenting until I'm sure...

Reply:
Yes, please disregard me. It looks like an issue within the rst itself which Felix seems to have already fixed here: https://github.com/ansible/ansible-documentation/commit/15ac6a05ed030d9ceaea0283e90abb96c0182dbe

Reply:
Yeah, the conditionals here are rather complex. I did the bare minimum to merge the config modules into one, but intentionally avoided any refactoring, even though, I there were certain known bugs present in some of those configs. I think those should be fixed separately for atomicity.